### ISSUE TITLE
possible issue with 3.1.0.1346 with PDF templates

### ISSUE THREAD

**Issue by burgessa23** [isaacwisdom/RemarkableLamyEraser#44]
Hi, I'm on the latest beta and just installed v2.
the basic functionality is working for hold button to erase, I'm having an issue with click and click and hold.
any multiple clicks or holds close my file and open the main menu as if its not registering left handed setup. 
any ideas?

config:
``` GNU nano 4.9.3                                                                                        .config/LamyEraser/LamyEraser.conf                                                                                                  
# RemarkableLamyEraser Configuration File~
# Set up your triggers and effects here
# Any trigger left blank will have no effect.
# Effects are divided into tools and actions :
# Actions are compatible with click-type actions

# The available actions are:
# toolbar         : presses the toolbar panel button
# writing         : presses the writing utensil button
# color-black     : presses the black color in the stroke panel
# color-grey	  : presses the grey color in the stroke panel
# color-white     : presses the white color in the stroke panel
# undo            : presses the undo button
# redo            : presses the redo button

# Tools are compatible with press&hold type triggers, or with click type triggers as toggles
# The available tools are:
# eraser          : changes to eraser tool. on deactivation, changes back to writing utensil
#                   note that on the RM2, this mode emulates the eraser side of the marker plus.
# erase-selection : changes to erase selection tool. on deactivation, changes back to writing utensil
# select          : changes to select tool. on deactivation, changes back to writing utensil

# Example valid configuration lines:
# click                     erase
# press&hold                select


click
double-click                undo
triple-click                writing
quadruple-click             toolbar
quintuple-click

press&hold                  eraser
double-press&hold           erase-selection
triple-press&hold           select
quadruple-press&hold
quintuple-press&hold



```

**Comment by burgessa23** [comment]
I restarted twice, and switched to right handed, then back to left and that seems to have fixed it

**Comment by burgessa23** [comment]
ok, after some more research it seems that over time in LH mode: 

- press&hold works as expected
- triple-press&hold results in undo firing
- double-press&hold fires select
- double-click seems to not fire anything
- triple and quadruple click for writing and toolbar both consistently work.

Everything appears to work normally immediately after restarting and changing handedness.

**Comment by isaacwisdom** [comment]
So I just did some texting and can confirm the results you were seeing. I've also figured out why. With the addition of the new typing tool in the 3.0 update, the icons on the left are no longer in the same place they used to be, but by chance, they haven't seemed to have moved enough to break the basic functionality of the tool. However, the text tool is *not* available in PDF documents, and does cause a misalignment.  I've taken a picture of a pdf document and a horizontally flipped pdf for comparison.

<img width="629" alt="Screenshot 2023-01-11 at 12 58 03 AM" src="https://user-images.githubusercontent.com/66286016/211687095-aa2bf8ba-a7b1-401b-8652-8f47033233b7.png">

Unfortunately, given the way that my tool works, this might be an unfixable bug. I'll have to do more research to see if there is some way of seeing what kind of document is currently open to try and make the code adjust automatically for the discrepancy.

**Comment by isaacwisdom** [comment]
Good news, there is a way to tell. 
`UUID=$(PID=`pidof xochitl`; ls -l /proc/$PID/fd 2> /dev/null | grep lock | egrep -v 'rm|xochitl/xochitl' | egrep '[0-9a-f]{8}-([0-9a-f]{4}-){3}[0-9a-f]{12}' -o); cat /home/root/.local/share/remarkable/xochitl/$UUID.content | grep fileType`

This mega command will tell you if the open document is a notebook or a pdf, so there is a way for me to implement a solution. In fact, a lot of the difficult code is already there, since this is basically the same process I use for checking the orientation. I'll have to think about the best way to implement the offset for pdf documents.

**Comment by burgessa23** [comment]
Saw you pushed a branch up. would you like me to test? I need to reinstall anyway as I just updated to 3.1.1 beta

**Comment by isaacwisdom** [comment]
No need quite yet! The new branch isn't finished yet, I'm just documenting the gradual changes towards a complete solution. I'll create another comment here once I have a commit ready for testing.

**Comment by sshiva01** [comment]
I'm having the same issue with pdfs, with the action for the eraser tool instead triggering the selection tool. Looking forward to the next version of this -- thanks for all the work! I like that I can just use this with my Scribe pen instead of going through the whole rigmarole of DDVK hacks that I don't quite need.

**Comment by isaacwisdom** [comment]
Ok! Some of the issues should be fixed as of [8b8f163](https://github.com/isaacwisdom/RemarkableLamyEraser/commit/8b8f163fd09586e789cdf72c161d2144247ebfa5). In particular, this should account for the new text tool in notebooks and its absence in PDFs. Try it out and let me know of any bugs you notice!
